cirq-google/cirq_google/engine/engine_job_test.py

```diff
@@ -550,6 +551,53 @@ def test_results_getitem(get_job_results):
         _ = job[2]
 
 
+def test_receives_results_via_stream_returns_correct_results():
+    qjob = quantum.QuantumJob(
+        execution_status=quantum.ExecutionStatus(state=quantum.ExecutionStatus.State.SUCCESS),
+        update_time=UPDATE_TIME,
+    )
+    response_future = duet.AwaitableFuture()
+    response_future.try_set_result(RESULTS)
```

Comment:
Instead of creating a future and immediately setting a result, you could call `duet.completed_future`:
```suggestion
    response_future = duet.completed_future(RESULTS)
```
Same thing in several places below.